lib/streamlit/elements/time_widgets.py

```diff
@@ -214,17 +228,17 @@ class TimeWidgetsMixin:
     def time_input(
         self,
         label: str,
-        value: TimeValue = None,
-        key: Optional[Key] = None,
-        help: Optional[str] = None,
-        on_change: Optional[WidgetCallback] = None,
-        args: Optional[WidgetArgs] = None,
-        kwargs: Optional[WidgetKwargs] = None,
+        value: TimeValue | None | str | "builtins.ellipsis" = Ellipsis,
```

Comment:
Why do we need `str` here as type?

Reply:
It's a leftover when I used "..." instead of `Ellipsis` as default type, it's updated now 👍